locales/fr/translation.json

```diff
@@ -1,25 +1,25 @@
 {
-	"Turnip Calculator": "Turnip Calculator",
-	"ACNH Turnip Calculator": "ACNH: Turnip Calculator",
-	"Buy Price": "Buy Price",
-	"Daily Price": "Daily Price",
-	"Average": "Average",
+	"Turnip Calculator": "Calculatrice Navets",
+	"ACNH Turnip Calculator": "ACNH: Calculatrice de prix des navets",
+	"Buy Price": "Prix D'achat",
+	"Daily Price": "Prix du jour",
+	"Average": "Moyen",
 	"Maximum": "Maximum",
 	"Minimum": "Minimum",
-	"Mon Tue Wed Thu Fri Sat": "Mon Tue Wed Thu Fri Sat",
-	"AM": "AM",
-	"PM": "PM",
-	"Clear All Data!": "Clear All Data!",
+	"Mon Tue Wed Thu Fri Sat": "Lun Mar Mer Jeu Ven Sam",
+	"AM": "Matin",
+	"PM": "APRÈS-MIDI",
+	"Clear All Data!": "Remettre À Zéro!",
 	"Usage": "Usage",
-	"buyPriceInfo": "- The <1>Buy Price</1> value is for your own island. It doesn't matter if you buy turnips on another island!",
-	"priceChangeInfo": "- Prices change <1>twice a day.</1> Be sure to log them. (We save your data in your device).",
+	"buyPriceInfo": "- Le <1>Buy Price</1> est celui de ta propre île. Ce n'a pas d'importance si tu as acheté des navets sur une autre île!",
+	"priceChangeInfo": "- Les prix changent <1>deux fois par jour.</1> N'oubliez pas de vérifier. (Nous sauvegardons vos données dans votre appareil).",
 	"Changelog": "Changelog",
```

Comment:
Was "Changelog" left untranslated, or is the French word the same? Not sure.

Reply:
We use the same word in French. =) (Anglicism)